### DISCUSSION TITLE
openDTU on Battery - 124 durch Fernsteuerung abgeschaltet

### DISCUSSION THREAD

**Discussion by Harry7707** [hoylabs/OpenDTU-OnBattery#1297] [Q&A]
Hallo liebe Community,
ich bin langsam am verzweifeln. Mein Set-Up : openDTU on Battery, HSM-2000, Shelly 3em pro.
Soweit funktioniert alles, aber die Meldung "124 durch Fernsteuerung abgeschaltet" kommt ständig,
und der Live-Daten "Balken" wird gelb/orange.

Beginn        Ende         ID        Meldung
---------------------------------------------------------------
12:02:14     12:02:14     1         Wechselrichter gestartet
12:04:15     12:04:30     124     Durch Fernsteuerung abgeschaltet
12:08:24     12:08:39     124     Durch Fernsteuerung abgeschaltet
12:09:12     12:09:27     124     Durch Fernsteuerung abgeschaltet

[openDTU_Log.txt](https://github.com/user-attachments/files/17179232/openDTU_Log.txt)

Ich habe das ganze mal mitgeschnitten, vielleicht hat jemand eine Idee woran das liegen könnte.

Log Datei anbei.

Schon mal vielen Dank an Euch.

**Comment by spcqike** [comment]
Da brauchst du nicht verzweifeln. Einfach den log lesen 

> 12:04:16.667 > [DPL::announceStatus] calculated limit is less than minimum power limit	
12:04:16.723 > [DPL::updateInverter] Stopping inverter...

**Reply by Harry7707** [reply]
@spcqike  Ahhh, ok. Vielen Dank für den Hinweis.
Mein Verbrauch geht nie unter 100 - 150 Watt.
Und das untere Limit kann man nicht unter 10 Watt stellen ? Versteh' ich nicht ?

**Reply by Harry7707** [reply]
![image](https://github.com/user-attachments/assets/04196893-7d82-4b06-8b0b-a2fdb7dcb1cc)
Verbrauch = 453,3 Watt
Limit gesetzt auf 448 Watt
Inverter = 0 Watt

Wie kann das sein ?

**Comment by schlimmchen** [comment]
> Und das untere Limit kann man nicht unter 10 Watt stellen ?

Was spielt das hier für eine Rolle?

> Wie kann das sein ?

Ja, wie kann denn das nur sein?!? So ein Mist....

Du hast ein 24V System und dein Inverter macht komplett bescheuerte Dinge: Beispiel: Der DPL setzt ein Limit von 156W, und im nächsten Durchlauf des DPL sieht man, dass der Inverter 312W produziert (glaubt zu produzieren). Dann beschließt der DPL ein Limit von 708W anzufordert, wird auch eingestellt, meldet der Inverter auch zurück, aber er behauptet er produziert nur 170W.

Der DPL kann hier nicht vernünftig arbeiten wenn der Inverter irgendeine Leistung einstellt bzw. glaubt einzustellen weil seine Shunts in Sättigung sind...

**Reply by Harry7707** [reply]
o.k. danke für die Diagnose, schlimmchen.
Und warum läuft das hier nicht richtig, warum macht mein Inverter komplett bescheuerte Dinge ?
Was kann ich tun ?

**Reply by spcqike** [reply]
Dein Problem ist komplex. In erster Linie wird es mit den 24V zusammen hängen. Die Erfahrung zeigt dass 24V Systeme oft bei 200-250W pro MPPT stagnieren. Mehr wird unsauber und macht Probleme. (Liegt am hohen Strom, die shunts im wechselrichter gehen irgendwann in die Sättigung) 

dazu kommt beim HMS-2000, dass er Leistungen < 300W nicht sauber ausregelt. 75W pro MPPT sind einfach zu wenig. 

Zuletzt kann bei dir ein „Ungleichgewicht“ der Zuleitungen dazu kommen. Wenn die Leitungen unterschiedlich lang sind oder unsauber gemacht, hohe Widerstände haben, fließt Strom von A über B zurück und das führt zu unsauberen regelverhalten. 

Erstes und zweites hast du bestimmt. Drittes kann ich so nicht beurteilen. Kann aber ebenfalls vorkommen.

**Reply by Harry7707** [reply]
Also bei 3. bin ich mir sicher, dass das "sauber" ist ( vielleicht wenige cm Unterschied )
Bei 1. könnte ich meine 2 Batterien ( 24 V ) in Reihe schalten, dann hätte ich ein 48 V System.
Bei 2. könnte ich nur z.B. nur 2 von 4 Eingängen benutzten, dann hätte ich 150 W pro MPPT ? Oder ich tausche den HMS 200 in einen z.B. 1000 Watt ?

**Reply by spcqike** [reply]
3. kannst du bewerten indem du einfach mal ein festes Limit einstellst und guckst, ob der WR bei allen Eingängen die gleiche Leistung meldet. Und natürlich ob am Ende das raus kommt, was raus kommen soll. 

48V sind natürlich in vielerlei Hinsicht besser, als 24V. 

Ob das Problem an 75W pro mppt liegt oder an 300W in Summe, habe ich so noch gar nicht getestet. Ich weiß nur, das mein hms-2000 bei <300W unsauber ist.

**Reply by Harry7707** [reply]
ok. Vielen Dank für die Einschätzungen.

**Reply by schlimmchen** [reply]
> Bei 1. könnte ich meine 2 Batterien ( 24 V ) in Reihe schalten, dann hätte ich ein 48 V System.

Du hast zwei Batterien und ziehst Reihenschaltung in Betracht? Super, dann solltest du das tun. :muscle: Ein 2kW WR bei einer 24V Batterie ist schlicht keine gute Idee. Dieser Umbau wird die Eigenheiten eines 24V Systems abstellen.

Kabel sollten ja hinreichend dick sein, du halbierst ja somit den Strom. Du musst lediglich darauf achten, dass dir die zwei Batterien dann nicht auseinander driften von der Ladung/Spannung her. Das musst du im Auge behalten. Könntest z.B. einen Smartshunt einsetzen, der kann auch eine "Midpoint Voltage" messen, die du dann in OpenDTU-OnBattery siehst.

> Bei 2. könnte ich nur z.B. nur 2 von 4 Eingängen benutzten, dann hätte ich 150 W pro MPPT ?

Der HMS-2000 hat 4 MPPTs, also einen pro Eingang. Jeder kommt irgendwo bei 250W in die Sättigung, d.h. der WR kann dann nicht mehr genau wissen, wie viel Strom er tatsächlich produziert, und dann gibt's Ungereimtheiten.

**Reply by spcqike** [reply]
> Jeder kommt irgendwo bei 250W in die Sättigung

hat das schon mal jemand getestet? Ich gehe vorsichtshalber zwar auch immer davon aus, aber der hms-2000 kann 16A pro Eingang. Das sollten bei 24V so 380W werden. 🤷‍♂️ hab nur keine Batterie, um das mal zu testen :D

**Reply by Harry7707** [reply]
Also String 1 und 2 haben 104 Watt und String 3 und 4 haben 122 Watt, obwohl der Unterschied der Kabellängen nur 40 cm beträgt...
Nur String 1 und 2 zu betreiben ( habe ich ausprobiert ) funktioniert mit dem DPL gar nicht.

Werde mir den Smartshunt besorgen, und auf 48 V umbauen.

to be continued...

**Reply by Harry7707** [reply]
Habe 2 von 3 möglichen Störfaktoren beseitigt.
1. die 4 Eingänge des WR sind nun fast gleich, Keine Differenzen mehr.
2. Habe den HMS 2000 in einen HMS 1600 getauscht ( dachte der könnte im unteren Bereich besser regeln ).

Diese beiden Maßnahmen haben auf die Regelung quasi keinen Einfluss. Die Regelung ist nach wie vor ziemlich chaotisch....

Was nun noch bliebe wäre die Umstellung von 24 V auf 48 V. Dazu müsste ich meine beiden 24 V Batterien in Reihe schalten.
Hierfür wäre ein Balancer und ein nicht unerheblicher Umbau mit Kosten von Nöten.
Ob das dann die Lösung ist ???

**Reply by spcqike** [reply]
> Ob das dann die Lösung ist ???

meiner Meinung nach nicht. Nicht langfristig. 

Du könntest sie kurzfristig auch ohne balancer und dergleichen betreiben. Nur um das „48V Feeling“ einmal zu testen. 

Langfristig würde ich mir eine 48V Batterie holen. Die vorhandenen? Entweder anderweitig mit 24V nutzen oder verkaufen.

**Comment by Harry7707** [comment]
> > Ob das dann die Lösung ist ???

> meiner Meinung nach nicht. Nicht langfristig.

Wenn das nicht die Lösung ist, was dann ? Und dann könnte ich mir das 48 V Feeling ja auch sparen :-)

**Reply by swingstate** [reply]
Ich hatte mal 2x 24V in Serie laufen inkl. Shunt und Balancer (den musst Du erstmal finden; muss 24V LFP balancen können). Das geht schon, ist aber relativ aufwendig die 2 Batterien hatten kein BMS mit dem man die Batterien überwachen konnte. Habe ich dann verkauft und 2x 48V Pylontech installiert.

**Reply by spcqike** [reply]
> was dann ?

neben Beitrag hast du schon bis zu Ende gelesen, oder?

**Reply by Harry7707** [reply]
Sorry. Ich fragte, ob das dann die Lösung ist ( Umstellung auf 48 Volt )
Deine Antwort "meiner Meinung nach nicht. Nicht langfristig"
Im letzten Satz schreibst Du aber dann "Langfristig würde ich mir eine 48V Batterie holen"

??

**Reply by spcqike** [reply]
Du hast gefragt 

> Was nun noch bliebe wäre die Umstellung von 24 V auf 48 V. Dazu müsste ich meine beiden 24 V Batterien in Reihe schalten.
Hierfür wäre ein Balancer und ein nicht unerheblicher Umbau mit Kosten von Nöten.
Ob das dann die Lösung ist ???

also ein 48V System aus zwei einzelnen 24V batterien. Das ist für mich keine langfristige Lösung.
 Eine einzelne 48V Batterie hingegen schon. Das sind für mich halt zwei unterschiedliche Herangehensweisen und total unterschiedliche Dinge. 
 
Was widerspricht sich dort oder ist nicht verständlich?

**Reply by Harry7707** [reply]
Ja. 'tschuldigung. Jetzt weiß ich wie du das meinst.
Werde das "provisorisch" ausprobieren, und dann vermutlich auf 48 V umsteigen.
Aber wie du schon sagst, mit EINER 48 V Batterie.

**Comment by Harry7707** [comment]
@swingstate  Ja, das mit den 2 x 24 V sehe ich genau so. Pylontech ist sicher eine gute Idee, oder auch Anker Solix.
Ich wollte aber nicht noch mehr Geld reinstecken.... Müsste mal versuchen meine beiden 24 V 100 Ah Redodo und den Victron 150 / 70
zu verkaufen.

Schade das der Open DTU nicht so wirklich funktioniert.

**Reply by schlimmchen** [reply]
> Schade das der Open DTU nicht so wirklich funktioniert.

Definitiv nicht OpenDTU's oder OpenDTU-OnBattery's Schuld. Hätte man auch vor dem Kauf der 24V Batterien schnell recherchieren können, auch wenn das noch nicht im Detail dokumentiert ist, dass die WR mit 24V nicht gut nutzbar sind.

**Comment by Harry7707** [comment]
Tja, meine Anlage ist halt mit der Zeit gewachsen :

Erst 2 Module 425 W ( ohne alles )
dann 1. Batterie 24 V 100 Ah mit WR fest eingestellt 175 Watt ( gedacht für die Nachteinspeisung )
dann noch 2 Module 425 Watt und 2. Batterie 24 V 100 Ah mit 2 Stufen-Schaltung des WR ( über Shelly ) 175 W und bei erhöhtem Verbrauch 380 Watt ( hat soweit gut funktioniert )
Wollte es dann aber noch "genauer" und mehr ( Einspeisung ) haben = OpenDTU.....

Und wie du schon gesagt hast, ich habe keine Dokumentation gefunden, in der man von einem 24 V System, für die Open DTU, abgeraten hätte ....

**Reply by Manos1966** [reply]
> Und wie du schon gesagt hast, ich habe keine Dokumentation gefunden, in der man von einem 24 V System, für die Open DTU, abgeraten hätte ....

"We agree to disagree" 😉 
![24V issues with hoymiles](https://github.com/user-attachments/assets/4355d9c6-0fbf-49b0-a5e3-8a5ed14beab2)

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/24V-Battery-issues-with-Hoymiles

**Reply by Harry7707** [reply]
Danke für den Hinweis. Werde dann wohl auf 48 V umsteigen.

**Reply by Harry7707** [reply]
Wäre vielleicht ganz gut, wenn der Verkäufer der OpenDTU darauf hinweisen würde,
dass ein 24 Volt System nicht so stabil läuft....

**Reply by spcqike** [reply]
> Wäre vielleicht ganz gut, wenn der Verkäufer der [...]

die openDTU ist erst einmal nur eine DTU, eine Data Transfer Unit. Sie kommuniziert mit dem Wechselrichter. 
Die Wechselrichter selbst sind vom Hersteller gar nicht für den Betrieb mit einer Batterie gedacht. 
Und: die "reine" openDTU ist für PV Systeme ohne Batterie.

Warum sollte der Verkäufer also auf die Probleme mit 24V Batterien hinweisen? Er weißt ja auch nicht darauf hin, dass das Gerät nicht unter Wasser betrieben werden kann, oder im All, oder sonst was für mögliche Szenarien, die man sich nur vorstellen kann.

**Comment by Harry7707** [comment]
@swingstate  wie zufrieden bist du denn mit dem Pylontech Speicher ?

**Reply by swingstate** [reply]
Läuft bestens. Gibt hier einige die auf diesen Speicher setzen.

**Comment by Harry7707** [comment]
@schlimmchen  das heißt ja aber im Umkehrschluss, dass, wenn ich auf 48 Volt umstelle, das Ganze gut funktionieren sollte ?

**Reply by schlimmchen** [reply]
Also mein System funktioniert super 👍

**Reply by frankydee1980** [reply]
Bin jetzt auch von 24V auf 48V umgestiegen. Es ist definitiv stabiler. Lohnt sich auf jeden Fall...

**Reply by Harry7707** [reply]
@frankydee1980   Danke für das Feedback.

**Comment by Harry7707** [comment]
Habe auf 48 V umgestellt. ( vorerst mit 2 x 24 V ).
Das ist, wie Ihr ja auch schon gesagt habt, eine ganz andere Regelung ! Viiiiiel besser.
Vielen Dank für Eure Hilfe.

![image](https://github.com/user-attachments/assets/c34f041a-fab7-4922-8605-e2e804dc4ff2)
